test/jdk/java/net/ProxySelector/SystemProxies.java

```diff
@@ -23,7 +23,7 @@
 
 /*
  * @test
- * @bug 6912868 8170868
+ * @bug 6912868 8170868 8316433
```

Comment:
You may have used this test to ensure that the change didn't cause a regression but I don't think it should be added the `@bug` list for the test.

Reply:
Thanks, I wasn't sure about that. I'll revert.